addons/web/models/ir_actions_server.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from uuid import uuid4
+from json import JSONDecodeError
+from secrets import compare_digest
+import hashlib
+
+from odoo.http import request
+from odoo import fields, models, api, exceptions, _
+
+
+class ServerAction(models.Model):
+    _inherit = "ir.actions.server"
+
+    url = fields.Char("Public Url", compute="_compute_url")
+    url_path = fields.Char("URL Path", help="Additional URL path", compute="_compute_url_path", store=True)
+    expose = fields.Selection([("webhook", "Expose as webhook")],
+        help="Expose this server action via a controller. The 'as webhook' option will make the server action available via '/web/hook/<url_path>'")
+
+    log_webhook_calls = fields.Boolean(string="Log Calls", default=True)
```

Comment:
Shouldn't it be default="False" ? or no default, as we only want it when debugging.

Reply:
yes it can